test/integration/docker_build_test.exs

```diff
@@ -104,6 +104,26 @@ defmodule ExDockerBuild.Integration.DockerBuildTest do
       assert log =~ "STEP 5/6 : VOLUME vol_storage:/myvol"
       assert log =~ "STEP 6/6 : CMD [\"cat\", \"/myvol/greeting\"]"
     end
+
+    test "create and manage persistent storage as volumes that can be attached to containers" do
+      volume_name = "vol_storage"
+      instructions = [
+        {"FROM", "alpine:3.8"},
+        {"VOLUME", volume_name},
+        {"VOLUME", "vol_storage:/myvol"}
+      ]
+
+      _log =
+        capture_log(fn ->
+          assert {:ok, image_id} = DockerBuild.build(instructions, "")
+        end)
+
+        {:ok, body} = ExDockerBuild.get_volumes(%{"name" => volume_name})
+        assert %{"Volumes" => [%{"Name" => volume_name, "Scope" => "local"}]} = body
+        {:ok, volume_body} = ExDockerBuild.inspect_volume(volume_name)
+        assert %{"Name" => volume_name, "Scope" => "local"} = volume_body
```

Comment:
this test is always going to pass because is overwriting the `volume_name` you should use `^volume_name` here so you assert the name is the same